### PR TITLE
Japi 279 use reponse wrapper

### PR DESCRIPTION
**Pull request by woodbr01**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [x] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [x] The change has been fully tested:
  - [x] I have performed unit tests to cover my changes.
  - [x] I have performed system test and covered possible regressions and side effects.
  - [x] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [x] Scalability
  - [x] Performance
  - [x] Security
  - [x] Thread-safety
  - [x] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.

## Testing:

Added unit tests

### CODE REVIEW

File: wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java [hunk 1]
```diff
@@ -187,10 +188,11 @@ public static DropZoneWrapper getDropzone(List<DropZoneWrapper> dropzones, Strin
         return null;
     }
 
+    @Category(IntegrationTest.class)
     @Test
     public void testUploadFile()
     {
-        File uploadFile = new File("src/test/resources/fileSprayTest/salesdata_orig.csv");
+        File uploadFile = new File("src/test/resources/fileSprayTest/Japi274deleteDropzoneFileTest.csv");
```
Comment (rpastrana) [inline wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java]: Does this file actually exist?

File: wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java [hunk 2]
```diff
@@ -264,19 +267,20 @@ public void testDeleteDropZoneFileInvalidFile()
...
             HPCCWsClient wsClient = platform.checkOutHPCCWsClient();
-            DFUWorkunitsActionResponse result = filesprayclient.deleteDropZoneFiles(dropzoneName, fileNames, wsClient.getHost(), path, null);
-            if (result.isExceptionsSpecified())
+            DFUWorkunitsActionResponseWrapper result = filesprayclient.deleteDropZoneFiles(dropzoneName, fileNames, wsClient.getHost(), path, null);
+
+            if (!result.getExceptions().getException().isEmpty())
             {
-                System.out.println("failed");
-                System.out.println(result.getFirstColumn());
+                Assert.fail("Got an exception when trying to delete the non existent file\n" + result.getExceptions().toString());
```
[the remark this answers is not in the document]
Comment (woodbr01) [reply wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java]: As long as everything else is correct (dropzone, username, etc) then yes no exception is thrown. It returns a DFUActionResult with a result string that reads "Warning: this file does not exist.". I check for this below on line 280.

File: wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java (hunk 1 above)
Comment (woodbr01) [reply wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java]: Yes. I re-named the file before submitting my last PR but didn't change the name here.

**Comment by rpastrana** [comment]
@woodbr01 looks good.

**Pull request merged**